src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -3410,7 +3410,7 @@ class C
         return x;
     }
 
-    private async Task<string> NewMethod()
+    private async Task<string?> NewMethod()
```

Comment:
❤️

Reply:
Sorry, just saw this! did you want me to make these changes and submit another pr since this one was merged? Thanks!